### ISSUE TITLE
Filter prioritization

### ISSUE THREAD

**Issue by zerotohero** [xteve-project/xTeVe#46]
Maybe there's a better way to do this, but I've found myself shuffling channels around a lot trying to get things just right. I like my channels to be grouped together logically, first by type then alphabetically. If I start moving channels around manually, then try to add an additional filter to add a new channel, it puts the new channels in the first available channel number and it just looks out of place in the guide. What I've found is that I can delete the stream and re-add it to let the filters re-group everything, but there doesn't seem to be any priority so the streams aren't added in the optimal order. Short of implementing a more robust channel organization feature, if we had a way to prioritize the order of the filters that would be a welcome intermediate step. So, for instance, upon getting a new list of streams, I could specify to have local news channels first, sports channels second, movie channels third, etc channels after. It wouldn't help with channel numbers getting changed around, but at least everything would be in a logical order without requiring additional manual intervention.

Or is there some other way to achieve a similar result that I'm unaware of?

Thank you, this is an amazingly useful project!

**Comment by mar-mei** [comment]
The order of the channels you have to put together yourself. Everyone has their own system, some want it alpabetic, others by category or country. xTeVe allows you to change the channel number. New channels always start at the first free channel position from 1000. If you change the channel numbers of your channels smaller than 1000, your order will remain. Pre-set your provider does not change the channels.

Example:
Channel 1-49 News
Channel 50-99 Sports
etc.

**Comment by zerotohero** [comment]
Yeah, that's why I was thinking filter prioritization could help re-organize channels. If it's any easier to implement, just being able to quickly enable or disable a filter the same way you can enable/disable a channel would also help. At the moment, you have to delete all of the filters and then re-create them to have the ordering regenerated in a specific order.

**Comment by MCTyler** [comment]
Or just order them the way you like in the first place ;)
